stable/kommander/README.md

```diff
@@ -27,3 +27,23 @@ helm install --namespace "kommander" --name "kommander-kubeaddons" --set kommand
 ```bash
 helm install --namespace "kommander" --name "kommander-kubeaddons" --set kommander-cluster-lifecycle.certificates.issuer.selfSigned=true ./stable/kommander
 ```
+
+## Backporting / Patch Releases
+The minor version should be bumped following every Kommander release.
+
+- Kommander v1.4.x -> chart version v0.14.x
```

Comment:
should the chart version here be `v0.16.x` to match the version in the chart above?

Reply:
yess.. lol this version has changed a lot in the lifetime of this pr 😅